src/pybind/mgr/dashboard/frontend/src/app/shared/components/pwd-expiry-notification/pwd-expiry-notification.component.ts

```diff
@@ -0,0 +1,61 @@
+import { Component, OnInit } from '@angular/core';
+
+import { forkJoin as observableForkJoin } from 'rxjs';
+
+import { UserFormModel } from '../../../core/auth/user-form/user-form.model';
+import { SettingsService } from '../../api/settings.service';
+import { UserService } from '../../api/user.service';
+import { AuthStorageService } from '../../services/auth-storage.service';
+
+@Component({
+  selector: 'cd-pwd-expiry-notification',
+  templateUrl: './pwd-expiry-notification.component.html',
+  styleUrls: ['./pwd-expiry-notification.component.scss']
+})
+export class PwdExpiryNotificationComponent implements OnInit {
+  alertType: string;
+  expiryDays: number;
+  levels = {};
+  private warningLevel = 'USER_PWD_EXPIRY_WARNING_1';
```

Comment:
Getting error when building:

```
ERROR in src/app/shared/components/pwd-expiry-notification/pwd-expiry-notification.component.html(1,56): Property 'warningLevel' is private and only accessible within class 'PwdExpiryNotificationComponent'.
```

Reply:
Thank you for your review @bk201 . And also sorry, I forgot to open the PR as a draft. I'll mark it as WIP.